README.md

```diff
@@ -117,6 +116,13 @@ The example may be run using:
     n_members -- size of the committee
     n_finalized -- number of data to be finalized
 
+The second example: `blockchain` is meant for benchmarking AlephBFT in the blockchain setting. It implements a simple round-robin blockchain assuming honest participation. The simplest way to run it is to use the provided script as follows (assuming we start in the root directory)
+
+```
+./examples/blockchain/run_blockchain 4
+```
+where `4` in the above is the number of committee members and can be replaced by any reasonable number. Running this script will result in generating log files `node0.log, node1.log, ...` corresponding to subsequent nodes. As one of the last log messages the achieved transactions per second count should be displayed.
```

Comment:
The last sentence has somewhat forced grammar, maybe "The achieved transactions per second should be among the final log messages in these files."?

Reply:
Yeah -- now better :)